### PR TITLE
401 use wftarget as coloured badge

### PR DESCRIPTION
**Pull request by wouter1975**
- Applies badge style to TARGET in workflow metadata table
- Applies badge style to TARGET in process/task list table

### CODE REVIEW

File: packages/orchestrator-ui-components/src/components/WfoBadges/WfoWorkflowTargetBadge/WfoWorkflowTargetBadge.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                return {
+                    badgeColor: toSecondaryColor(primary),
+                    textColor: primaryText,
+                };
+        }
+    };
+
+    const { badgeColor, textColor } = getBadgeColorFromTarget(target);
+
+    if (target !== null) {
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoBadges/WfoWorkflowTargetBadge/WfoWorkflowTargetBadge.tsx:58]: Small style preference. I would to the nullCheck on the target on line 13 so you bail out of the function immediately when there is not target. So 

```if(!target) return null```

**Review by DutchBen — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoBadges/WfoWorkflowTargetBadge/WfoWorkflowTargetBadge.tsx (hunk 1 above)
Comment (wouter1975) [reply packages/orchestrator-ui-components/src/components/WfoBadges/WfoWorkflowTargetBadge/WfoWorkflowTargetBadge.tsx]: Thanks, I updated code.

File: packages/orchestrator-ui-components/src/components/WfoBadges/WfoWorkflowTargetBadge/WfoWorkflowTargetBadge.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                return {
+                    badgeColor: toSecondaryColor(primary),
+                    textColor: primaryText,
+                };
+        }
+    };
+
+    const { badgeColor, textColor } = getBadgeColorFromTarget(target);
+
+    if (!target) return null;
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoBadges/WfoWorkflowTargetBadge/WfoWorkflowTargetBadge.tsx:58]: I would move this null check up to line 13 so you bail out right away if there is no target

**Review by DutchBen — DISMISSED** [review]

File: packages/orchestrator-ui-components/src/components/WfoBadges/WfoWorkflowTargetBadge/WfoWorkflowTargetBadge.tsx (hunk 2 above)
Comment (wouter1975) [reply packages/orchestrator-ui-components/src/components/WfoBadges/WfoWorkflowTargetBadge/WfoWorkflowTargetBadge.tsx]: check!

**Review by DutchBen — APPROVED** [review]

**Pull request merged**